localstripe/resources.py

```diff
@@ -1527,19 +1530,30 @@ def __init__(self, url=None, limit=None):
         self.url = url
 
         self._limit = limit
+        self._starting_after = starting_after
+        self._starting_pos = 0
```

Comment:
This variable looks inconsistent: if `has_more()` is called before `data()`, it will have a wrong value.